src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.links.strings

```diff
@@ -41,3 +41,5 @@
 "Test_Centers_FAQ_Link" = "https://www.coronawarn.app/de/faq/#where_can_i_get_tested";
 
 "HealthCertificate_InvalidSignature_FAQLink" = "https://www.coronawarn.app/de/faq/#hc_signature_invalid";
+
+"Statistics_Info_Blog_Link" = "https://www.coronawarn.app/en/blog/2021-08-05-statistictiles";
```

Comment:
en in de?

Reply:
Please change it to "https://www.coronawarn.app/de/blog/2021-08-05-statistictiles"

Reply:
Changed en link to de for de.
Added en link for all others.